.buildkite/pipeline.yml

```diff
@@ -99,50 +99,52 @@ steps:
     concurrency: 5
     concurrency_group: 'browserstack-app'
 
-  - label: ':android: Android 6 end-to-end tests'
+  - block: "Trigger full test suite"
```

Comment:
I think that block will mean that this doesn't technically pass from a github p of v until it is unblocked?

Why do the tests take 30 minutes to run?

Reply:
This is desired behaviour, as we can merge to non-protected branches but should run the full suite when merging to next/master.

Reducing CI job time is being discussed separately and is out-of-scope for this PR